### PR TITLE
Enable attendance confirmation if registered

### PR DESCRIPTION
**Pull request by littc**

### CODE REVIEW

File: lib/Zureg/Main/Lambda.hs [hunk 1]
```diff
@@ -103,22 +103,23 @@ main = do
...
-            -}
-
+                registrant <- Database.getRegistrant db uuid
+                case rState registrant of 
+                  Just Registered -> do Database.writeEvents db uuid [Confirm]
+                                        return $ Serverless.response302 $
+                                          "ticket?uuid=" <> E.uuidToText uuid
+                  _               -> return $ Serverless.response302 $
+                                          "ticket?uuid=" <> E.uuidToText uuid
+
```
Comment (jaspervdj) [inline lib/Zureg/Main/Lambda.hs:114]: You can take out the common bit of code here if you want:

```haskell
registrant <- Database.getRegistrant db uuid
case rState registrant of 
  Just Registered -> Database.writeEvents db uuid [Confirm]
  _               -> return ()
return $ Serverless.response302 $ "ticket?uuid=" <> E.uuidToText uuid
```

**Review by jaspervdj — COMMENTED** [review]
Looks good, one small comment and some notes that can be removed now.

File: lib/Zureg/Views.hs [hunk 2]
```diff
@@ -136,20 +136,18 @@ ticket r@Registrant {..} = template
                     H.! A.value "Take me back to the registration"
 
         {-
-
         NOTE (jaspervdj): The following snippet of code shows the "confirmation"
         button.  We will want to enable this as soon we as we send out the
         confirmation reminder to all attendees.
```
Comment (jaspervdj) [inline lib/Zureg/Views.hs]: Feel free to remove this note

File: lib/Zureg/Main/Lambda.hs [hunk 3]
```diff
@@ -103,22 +103,23 @@ main = do
...
                     registrant <- Database.getRegistrant db uuid
                     html $ Views.scan registrant
-
+                    
             {-
-
             NOTE (jaspervdj): We only want to enable this confirmation button
             handler when send out the confirmation email.
 
             TODO (jaspervdj): Check that they are not waitlisted?
```
Comment (jaspervdj) [inline lib/Zureg/Main/Lambda.hs]: Feeel free to delete these two notes

**Comment by littc** [comment]
Thanks for the review, Jasper. I have incorporated your comments.

**Review by jaspervdj — APPROVED** [review]

**Pull request merged**